packages/wp-now/src/wp-now.ts

```diff
@@ -141,23 +186,33 @@ export default async function startWPNow(
 	}
 
 	await installationStep2(php);
+
 	await login(php, {
 		username: 'admin',
 		password: 'password',
 	});
 
+	const initialCookies = php.requestHandler.serializeCookies().split(';');
```

Comment:
I'm getting an error that stops me to run `wp-now start`.
@seanmorris , is there something I'm doing wrong?

Here is my error:

```
node dist/packages/wp-now/cli.js start                                                                                   Node 20.8.1 13:44:19
Starting the server......
directory: /Users/macbookpro/proyectos/a8c/playground-tools
mode: playground
php: 8.0
wp: latest
WordPress latest folder already exists. Skipping download.
SQLite folder already exists. Skipping download.
TypeError: php.requestHandler.serializeCookies is not a function
    at startWPNow (file:///Users/macbookpro/proyectos/a8c/playground-tools/dist/packages/wp-now/main.js:851:45)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async startServer (file:///Users/macbookpro/proyectos/a8c/playground-tools/dist/packages/wp-now/main.js:1118:48)
    at async Object.handler (file:///Users/macbookpro/proyectos/a8c/playground-tools/dist/packages/wp-now/main.js:1325:25)
Failed to start the server: php.requestHandler.serializeCookies is not a function
```

**Test instructions**
1. Run `nvm use v20`
2. Run `npx nx build wp-now`
3. Run `node dist/packages/wp-now/cli.js start`
4. Observe the mentioned error

Reply:
@sejas We just deployed a new version of `wordpress-playground` that exposes the `serializeCookies` and `exit` methods, which were previously private. Try updating that package and see if the error goes away. Sorry if I'm less than responsive today, its a holiday in the states and I've got to run to see family.

Reply:
The new packages aren't released yet but I just triggered the release workflow so it should be soon.

It sounds like this PR should ship a package.json update to use the latest packages? Although that might depend on https://github.com/WordPress/playground-tools/pull/116 which may need to land first.

Reply:
@adamziel Thanks for chiming in! I had assumed that merging into master would publish an update automatically. In the meantime, is it possible to pull directly from `trunk` via npm? If not, I know its possible to use `npm link` for that.

Reply:
Publishing to trunk doesn't release new packages ATM. There's a GitHub action we're running manually from time to time.

Reply:
@adamziel I noticed you published a new version! I wish you'd told me! I'd have gotten these linter errors out of the way immediately. Thank you for getting that going though, I jumped on this as soon as I noticed.

Reply:
>  I noticed you published a new version! I wish you'd told me! 

I published the packages two days ago and let know on slack yesterday.

Reply:
@adamziel Not sure how I missed that one, but I'm glad we got it published!